### ISSUE TITLE
Benchmark comparison between Fairlearn's Reductions algorithm and FairTorch's  Constraint algorithm.

### ISSUE THREAD

**Issue by LeJit** [fairlearn/fairlearn#830]
To understand how [FairTorch](https://github.com/wbawakate/fairtorch)'s algorithm compares to FairLearn's Reduction algorithm, we want to create a workflow to compare the two algorithms on the same dataset.

## Process 

1.) Choose some publicly available datasets (e.g. Adult, German Credit).
2.) Run both Fairlearn's ExponentiatedGradient model (w/ DemographicParity Moment) and a PyTorch model with FairTorch's DemographicParityLoss constraint on the dataset.
3.) Compare the performance of both algorithms, in terms of stability of results, run-time, and correctness of end result.
4.) *End Deliverable*: A Python script or Jupyter notebook that executes the above workflow.

**Comment by adrinjalali** [comment]
Could you please give some motivation as why we need to run this comparison in particular, as opposed to comparing with other solutions?

**Comment by LeJit** [comment]
@romanlutz and I had a conversation with the FairTorch team last week, and we were discussing the possibility of integrating their algorithm into our `Reductions` module. 

However, it was not clear how their algorithm compares with Fairlearn's, so we want to run some comparisons to understand the differences between the two methods.

**Comment by adrinjalali** [comment]
Right, I understood the issue as adding the comparison to the docs, not as a task. All good :+1:

**Comment by TamaraAtanasoska** [comment]
@romanlutz and @LeJit is this benchamark still interesting to know a few years later?

**Comment by romanlutz** [comment]
If one had infinite time, sure, but otherwise no. As you can see on this thread, nobody has even asked about it in almost 4 years. Feels safe to close.
